gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
Gulp defines *dependencies*. Both `mocha` and `build` *depend* on `scripts`, so both should declare those as a dependency. gulp is smart enough to de-dupe that.

```JS
var gulp = require('gulp');

gulp.task('a', function() {
  console.log('Task a');
});

gulp.task('b', ['a'], function() {
  console.log('Task b');
});

gulp.task('c', ['a'], function() {
  console.log('Task c');
});

gulp.task('d', ['a', 'b', 'c'], function() {
  console.log('Task d');
});
```

```
$ gulp d
[15:27:32] Using gulpfile ~/src/scratch/syncspace/gulpfile.js
[15:27:32] Starting 'a'...
Task a
[15:27:32] Finished 'a' after 89 μs
[15:27:32] Starting 'b'...
Task b
[15:27:32] Finished 'b' after 51 μs
[15:27:32] Starting 'c'...
Task c
[15:27:32] Finished 'c' after 34 μs
[15:27:32] Starting 'd'...
Task d
[15:27:32] Finished 'd' after 23 μs
```